pom.xml

```diff
@@ -412,6 +412,30 @@ SOFTWARE.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>nullfree</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.nikialeksey</groupId>
+            <artifactId>nullfree-maven-plugin</artifactId>
+            <version>1.4.3</version>
+            <configuration>
+              <skipComparisions>true</skipComparisions>
+              <threshold>13</threshold>
```

Comment:
@nikialeksey How did you figure out the exact threslhold value??? Does it fail for 12 ???

Reply:
Yes, for now it fails with 12. That means takes contains 12 `null` without comparision context (cases as `if (obj == null)` does not included because of `skipComparision = true`).